README.md

```diff
@@ -69,8 +165,40 @@ Use the following arguments to modify the default values of the`agent.conf` file
     <input_file_path_1> <input_file_path_2> <input_file_path_n>
 ```
 Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
-## TODOs
 
-- for metrics agent:
-    - create service files
+### Monasca-ui
 
+The monasca-ui plugin can be installed via the following command:
+```
+./monasca-ui-<version>.run --target <monasca_ui_dir>
+```
+
+This will extract the plugin with all the required dependencies.
+
+Then it is necessary to perform a set of manual configuration steps.
+First of all, you need to append the monasca-ui virtualenv libraries to
+the Horizon system path. You need to find your wsgi script for Horizon 
+and edit it adding:
+
+```
+sys.path.append("<monasca_ui_dir>/lib/python2.7/site-packages/") 
+```
+
+Then it is required to enable the monasca-ui plugin. Simply create 
+symbolic links in horizon installation pointing to the monasca-ui
+installation:
+
+```
+ln -s <monasca_ui_dir>/lib/python2.7/site-packages/monitoring/enabled/_50_admin_add_monitoring_panel.py \
+      <horizon_dir>/openstack_dashboard/enabled/_50_admin_add_monitoring_panel.py
+ln -s <monasca_ui_dir>/lib/python2.7/site-packages/monitoring/conf/monitoring_policy.json \
+      <horizon_dir>/openstack_dashboard/conf/monitoring_policy.json
```

Comment:
First symbolic link creation works fine, but monitoring_policy.json already exists - identical to target